docs/expansions_and_fields.rst

```diff
@@ -40,6 +41,19 @@ using this parameter:
 * Attached place’s object
 * Any referenced Tweets’ object
 
+For methods that returns DirectMessageEvents, the following data objects can be
+expanded using this parameter:
+
+* ``sender_id`` - The ID of the account that sent the message, or who invited
+a participant to a group conversation
+* ``partricipants_ids`` - An array of account IDs. For ParticipantsJoin and
+ParticipantsLeave events this array will contain a single ID of the account
+that created the event
+* ``attachments`` - Provides media IDs for content that has been uploaded to
+Twitter by the sender
+* ``referenced_tweets`` - If a Tweet URL is found in the text field, the ID of
+that Tweet is included in the response
```

Comment:
I have plans to improve this section of the documentation in the future to include information like this from https://developer.twitter.com/en/docs/twitter-api/expansions, but for now, for consistency, this section should also be taken from the description for the expansions parameter for endpoints that return the object type, e.g. https://developer.twitter.com/en/docs/twitter-api/direct-messages/lookup/api-reference/get-dm_conversations-with-participant_id-dm_events for DM events.

```suggestion
For methods that returns Direct Message conversation events, the following data
objects can be expanded using this parameter:

* The user object for the message sender.
* Attached media's object.
* Any referenced Tweet's object.
* The user object for who is joining or leaving group conversations.
```

This should probably also be after the following section on user objects.

Reply:
I've added the tables from https://developer.twitter.com/en/docs/twitter-api/expansions now, with https://github.com/tweepy/tweepy/commit/7df645328bdf26e5c4c569392d5d2252b1527477.